extensions/github-repository-search/src/useRepositories.ts

```diff
@@ -53,7 +53,9 @@ export function useRepositories(searchText: string | undefined) {
       setState((oldState) => ({ ...oldState, isLoading: true }));
 
       try {
-        const { search } = await octokit.graphql<SearchRepositoriesResponse>(SEARCH_REPOSITORIES_QUERY, { searchText });
+        const { search } = await octokit.graphql<SearchRepositoriesResponse>(SEARCH_REPOSITORIES_QUERY, {
+          searchQuery: `${searchFilter} fork:true ${searchText}`,
```

Comment:
Should it be a preference if you want to include forks in the search? It's a bit confusing that you have `searchText` and `searchFilter`. Wouldn't it be easier to inject the entire `searchQuery` from outside the hook?

Reply:
Sure, can do it